Sources/Scan/DIScan.swift

```diff
@@ -14,12 +14,12 @@ public protocol DIScanned {}
 open class DIScan {  
   static func types(_ valid: (AnyClass)->Bool, _ predicate: (AnyClass)->Bool, _ bundle: Bundle?) -> [AnyClass] {
     if let bpath = bundle?.bundlePath {
-      return scanned.flatMap { cls, clsBPath in
+      return scanned.compactMap { cls, clsBPath in
```

Comment:
#if swift(>= 4.1)
#else
#endif
or for simple change:
```
#if swift(>= 4.1)
#else
extension Sequence
{
  func compactMap(...) { ... )
}
#endif
```

Reply:
Fixed